chcemvediet/apps/inforequests/templates/admin/inforequests/branch/delete_selected_confirmation.html

```diff
@@ -1,27 +0,0 @@
-{% extends "admin/delete_selected_confirmation.html" %}
```

Comment:
@viliambalaz Nie je dobre, ze kod:
```
    {% if snoozed_actions %}
      {% prepend path=".//form" %}
        <p>
          <label><input name="snooze" type="checkbox" /> Extend snooze of previous actions by
            {{ ADMIN_EXTEND_SNOOZE_BY_DAYS }} days from today? All of the following actions will be
            snoozed:
          </label>
        </p>
        <ul>{{ snoozed_actions|unordered_list }}</ul>
      {% endprepend %}
    {% endif %}
```
sa nachadza v globalnych `inforequests/delete_confirmation.html` a `inforequests/delete_selected_confirmation.html`. Takto sa tento kod aplikuje pre vsetky modely, aj pre tie, pre ktore nedava zmysel. Je sice pravda, ze pre ine modely nemame definovanu premennu `snoozed_actions`, tak kod sa pre ne nevykona. Ale nie je dobre rozne kod pre rozne modely takto prevazovat. Vznika z toho spageti kod. Co ak casom budeme mat inu premennu s rovnakym nazvom ale v inom modeli? Potom to zahadne zacne robit blbosti.

Namiesto globalnych `inforequests/delete_confirmation.html` a `inforequests/delete_selected_confirmation.html` nechaj radsej povodne `inforequests/*/delete_confirmation.html` a `inforequests/*/delete_selected_confirmation.html` zvlast definovane pre kazdy model. Akurat aby si nemusel ich obsah copy&pastovat, opakujuci kod vysun do pomocnych suborov a tie potom v tychto suboroch includni. Cize nakonci napriklad `inforequests/action/delete_confirmation.html` bude vyzerat cca takto:

```
{% block content %}
  {% amend %}
    {{ block.super }}
    {% include "admin/inforequests/mixins/delete_nested_inforequest_email_mixin.html" %}
    {% include "admin/inforequests/mixins/snoozed_actions_mixin.html" %}
    {% include "utils/admin/bulk_delete_mixin.html" %}
  {% endamend %}
{% endblock %}
```

A analogicky v suboroch pre ostatne modely includnes iba to, co sa ich tyka.

--
*Poznamka: Komentar je zaveseny k inemu suboru, kedze k `inforequests/delete_confirmation.html` ani `inforequests/delete_selected_confirmation.html` sa nedaju pisat komentare, kedze v nich nedoslo k zmenam.*

Reply:
@martinmacko47 , ja som to skusal aj predtym aby bol spolocny template, ale sprava sa to nekonzistente. Je potrebne aby bol `prepend`, `after`, `before`, ... v spolocnom template ako je `amend`. Problem mi teraz tiez nastava pre model `InforequestEmail`, ked nefunguje spravne `set_attributes`, submit button pre potvrdenie vymazania ostava aktivny. Zauimave ze pre model `Action` aj `Branch` sa podla ocakavania zamrazi. Tak necham to na teba na posudenie.